importer/importer_utils.py

```diff
@@ -11,7 +11,21 @@
 
 site_cache = {}
 
-DATA_DIRECTORY = "data"
+
```

Comment:
If you want to be on the safe side I would remake this as `get_data_directory(dir_name="data")` then call that in `nature_importer.py`, `reserve_harvester.py` and `importer_utils.py` where you previously used `DATA_DIRECTORY`